micrometer-core/src/test/java/io/micrometer/core/instrument/MissingLatencyUtilsTest.java

```diff
@@ -45,4 +52,16 @@ void createRecordCloseTimerWithoutPauseDetector() {
         }).doesNotThrowAnyException();
     }
 
+    @Test
+    @Issue("6863")
+    void timerWithPauseDetector_noExceptionThrown(CapturedOutput output) {
+        registry.config().pauseDetector(new ClockDriftPauseDetector(Duration.ofMillis(100), Duration.ofMillis(100)));
+        assertThatCode(() -> {
+            Timer timer = Timer.builder("my.timer").register(registry);
+            timer.record(1, TimeUnit.MILLISECONDS);
+            timer.close();
+        }).doesNotThrowAnyException();
```

Comment:
Before the changes in AbstractTimer, this test would throw a CFNE when trying to instantiate `SimplePauseDetector` if LatencyUtils is not on the classpath.